### PR TITLE
Add FOR loop for finding related bills

### PR DESCRIPTION
**Pull request by reginafcompton**
The newly added code in this PR iterates over matter relations (should they exist) and creates related bills. It assumes that each related bill is a "companion" - however, that may require further conversation. 

Related to [this issue](https://github.com/datamade/la-metro-councilmatic/issues/158) on Metro Councilmatic.

### CODE REVIEW

File: lametro/bills.py [hunk 1]
```diff
@@ -155,6 +158,24 @@ def scrape(self, window=28) :
             for topic in self.topics(matter_id) :
                 bill.add_subject(topic['MatterIndexName'].strip())
 
+            for relation in self.relations(matter_id):
+                try:
+                    # Get data (i.e., json) for the related bill. 
+                    # Then, we can find the 'MatterFile' (i.e., identifier) and the 'MatterIntroDate' (i.e., to determine its legislative session).
+                    # Sometimes, the related bill does not yet exist: in this case, throw an error, and continue.
+                    related_bill = self.endpoint('/matters/{0}', relation['MatterRelationMatterId'])
```
Comment (fgregg) [inline lametro/bills.py]: The checking of existence seems like something you might want to push upstream to python-legistar.

Comment (reginafcompton) [reply lametro/bills.py]: Okay....so, we could add a try-except block in the `endpoint` function: https://github.com/opencivicdata/python-legistar-scraper/blob/master/legistar/bills.py#L233

But I wonder if this is a "better-when-fragile" scenario? (At least, with the Metro bill relations, we know that the endpoint function often throw an error...but in other cases, maybe we want to see that error?)

Comment (fgregg) [reply lametro/bills.py]: That's a fair argument. If we find that we always want to skip the cases where an error is thrown, then we can move it to python-legistar.

**Comment by reginafcompton** [comment]
@hancush - scraper enhancement for Metro, FYI

**Pull request merged**